Stylish devdocs (#478)
OffchainLabs/arbitrum-docs: arbitrum-docs/stylus/partials/_stylus-public-preview-banner-partial.md

:::info PUBLIC PREVIEW CAPABILITY

> Stylus is currently a **[public preview](/stylus/concepts/public-preview-expectations)** `alpha` capability. The code has not been audited, should **not be used in production scenarios**.

Reviewer: ```suggestion
Stylus is currently tagged as an `alpha` release. The code has not been audited, and should **not be used in production scenarios**.
```

> To provide feedback, click the _Request an update_ button at the top of this document, [join the Arbitrum Discord](https://discord.gg/arbitrum), or reach out to our team directly by completing [this form](http://bit.ly/3yy6EUK).

Reviewer: ```suggestion
The Stylus docs are in a **[public preview](/stylus/concepts/public-preview-expectations)** capability. To provide feedback, click the _Request an update_ button at the top of this document, [join the Arbitrum Discord](https://discord.gg/arbitrum), or reach out to our team directly by completing [this form](http://bit.ly/3yy6EUK).
```
